pkg/kafka/topic/validators.go

```diff
@@ -45,6 +46,23 @@ func ValidateName(val interface{}) error {
 	return fmt.Errorf(`invalid topic name "%v"; only letters (Aa-Zz), numbers, "_", "." and "-" are accepted`, name)
 }
 
+func ValidateSearchInput(val interface{}, localizer localize.Localizer) error {
```

Comment:
Could you just re-use `ValidateName`?

Reply:
what if the user would want to filter with "." or ".."?

Reply:
Good point.